src/components/Nav/Nav.scss

```
@@ -0,0 +1,107 @@
+@import '../../styles/baseStyle/font.scss';
+@import '../../styles/baseStyle/size.scss';
+@import '../../styles/baseStyle/colors.scss';
+
+.navHead {
+  box-shadow: 0 3px 7px 0 rgb(0 0 0 / 10%);
+  .navUpper {
+    height: 40px;
```

```suggestion
.navHead {
  box-shadow: 0 3px 7px 0 rgb(0 0 0 / 10%);
  
  .navUpper {
    height: 40px;
```
새로운 셀렉터 나올 때마다 한줄 씩 띄워주세요

---

확인하여 적용되었습니다!